tests/api-diff/run.t

```diff
@@ -35,24 +35,27 @@ Here we generate a basic `.mli` files with two types and a function:
 
 ### A file with an additional type:
 
-  $ cat > add_type.mli << EOF
+  $ cat > add_type.mli <<EOF
   > type t = int
   > type added_t = float
   > val f : t -> string
-  > EOF 
-
+  > EOF
+ 
```

Comment:
Can you remove this trailing whitespace please?

You probably want to configure your editor to warn you about trailing whitespaces.

Reply:
Thank you for the suggestion. For anyone who might experience the trailing whitespaces issue too, here is  a script you can use in your editor to highlight, match and remove white spaces before saving to buffer.

```
highlight ExtraWhitespace ctermbg=red guibg=red
match ExtraWhitespace /\s\+$/
autocmd BufWritePre * %s/\s\+$//e
```